fsmonitor.c

```diff
@@ -147,6 +147,7 @@ void write_fsmonitor_extension(struct strbuf *sb, struct index_state *istate)
 static int query_fsmonitor(int version, const char *last_update, struct strbuf *query_result)
 {
 	struct child_process cp = CHILD_PROCESS_INIT;
+	int result;
```

Comment:
If you use `git commit --fixup <commit>`, it will use the syntax that `git rebase --autosquash` understands...

Reply:
(You could also simply replace the colon by an exclamation point in the commit message)